.github/workflows/build-release.yml

```diff
@@ -0,0 +1,86 @@
+# Build docker image, build cumulocity archive and release it to the repository
+
+name: Build & release oee-simulators
+
+on:
+  workflow_dispatch:
+    inputs:
+        release_body:
+          type: string
+          description: GitHub Release Message
+          default: "GitHub Release Message"
+  push: 
+    branches: [ COE-722 ]
+
+env:
+  BUILD_OFFSET: 25
+
+jobs:
+  build_upload:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v2
+        with:
+          path: 'repo'
+
+      - name: update versionnumber
+        run: |
+          cd repo/event-based-simulators
+          SIMULATOR_VERSION=$(expr ${{ env.BUILD_OFFSET }} + ${GITHUB_RUN_NUMBER})
+          sed -i "s/@project.version@/1.0.${SIMULATOR_VERSION}/g" cumulocity.json
+          echo "RELEASE_VERSION=${SIMULATOR_VERSION}" >> $GITHUB_ENV
+
+      - name: Zip the source-code
+        run: |
+          zip -r source-code.zip repo
+
+      - name: build docker image
+        run: |
+          cd repo/event-based-simulators
+          docker build -t oee-simulators .
+          docker save -o image.tar oee-simulators
+          docker rmi oee-simulators
+
+      - name: Zip it!
+        run: |
+          cd repo/event-based-simulators
+          zip oee-simulators.zip image.tar cumulocity.json
+        
+      - name: upload Zip
+        uses: actions/upload-artifact@v2
+        with:
+          name: 'oee-simulators'
+          path: 'repo/event-based-simulators/oee-simulators.zip'
+
+      - name: Create Release
+        uses: actions/create-release@v1
+        id: create_release
+        with:
+          draft: true
+          prerelease: false
+          release_name: v1.0.${{ env.RELEASE_VERSION }}
+          tag_name: v1.0.${{ env.RELEASE_VERSION }}
+          body: ${{ github.event.inputs.release_body }}
+        env:
+          GITHUB_TOKEN: ${{ github.token }}
+
+      - name: upload oee-sim zip
+        uses: actions/upload-release-asset@v1
+        env:
+          GITHUB_TOKEN: ${{ github.token }}
+        with:
+          upload_url: ${{ steps.create_release.outputs.upload_url }}
+          asset_path: 'repo/event-based-simulators/oee-simulators.zip'
+          asset_name: 'oee-simulators.zip'
+          asset_content_type: application/zip
+
+      - name: upload source-code zip
+        uses: actions/upload-release-asset@v1
+        env:
+          GITHUB_TOKEN: ${{ github.token }}
+        with:
+          upload_url: ${{ steps.create_release.outputs.upload_url }}
+          asset_path: 'source-code.zip'
+          asset_name: 'Source Code (zip)'
```

Comment:
maybe name this 'oee-simulators-sources.zip' oder 'oee-simulators-src.zip'?

Reply:
Done